packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -137,6 +137,9 @@ done
 %{_sbindir}/foreman-proxy-certs-generate
 
 %changelog
+* Wed Nov 06 2019 b'Zach Huntington-Meath <zhunting@redhat.com>' - b'1:1.22.2-1'
```

Comment:
not sure where these `b'` came from but looks like they are causing the build to fail (also in all other specs except foreman.spec)